esp32_sensors_node:
  ros__parameters:
    esp_port: '/dev/serial/by-id/usb-1a86_USB_Serial-if00-port0'
    esp_baudrate: 115200